### PR TITLE
Add dev.bambosh.UnofficialHomestuckCollection

### PR DESCRIPTION
**Pull request by Grafcube**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.   **Bambosh/unofficial-homestuck-collection#470**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *No patches necessary*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: LICENSE
Comment (hfiguiere) [inline LICENSE:1]: this is unecessary

File: dev.bambosh.UnofficialHomestuckCollection.desktop [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=The Unofficial Homestuck Collection
+Comment=An offline collection of Homestuck and its related works
+Exec=/app/bin/run.sh
```
Comment (hfiguiere) [inline dev.bambosh.UnofficialHomestuckCollection.desktop:6]:
```suggestion
Exec=run.sh
```

File: dev.bambosh.UnofficialHomestuckCollection.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```
Comment (hfiguiere) [inline dev.bambosh.UnofficialHomestuckCollection.yml:12]: does this really support wayland out of the box?

File: dev.bambosh.UnofficialHomestuckCollection.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      - cp -ar unofficial-homestuck-collection-${pkgver} ${FLATPAK_DEST}/unofficial-homestuck-collection
+      - |
+        for i in 16x16 24x24 32x32 48x48 64x64 128x128 256x256 512x512; do
+          install -Dm644 build/icons/$i.png ${FLATPAK_DEST}/share/icons/hicolor/apps/$i/dev.bambosh.UnofficialHomestuckCollection.png
+        done
+      - install -Dm644 dev.bambosh.UnofficialHomestuckCollection.metainfo.xml /app/share/metainfo/dev.bambosh.UnofficialHomestuckCollection.metainfo.xml
+      - install -Dm755 run.sh ${FLATPAK_DEST}/bin
+    sources:
+      - type: file
+        url: https://github.com/Bambosh/unofficial-homestuck-collection/releases/download/2.0.5/unofficial-homestuck-collection-2.0.5.tar.gz
```
Comment (hfiguiere) [inline dev.bambosh.UnofficialHomestuckCollection.yml:36]: do I understand this is a x86_64 binary? how will that work on aarch64?

**Review by hfiguiere — CHANGES_REQUESTED** [review]
have the .desktop and appstream file been submitted for inclusion upstream?

File: LICENSE
Comment (Grafcube) [reply LICENSE:1]: Done. I've removed the license file.

**Comment by Grafcube** [comment]
> have the .desktop and appstream file been submitted for inclusion upstream?

I didn't know those had to be upstreamed. I'll make the PRs.

**Comment by GiovanH** [comment]
TUHC member here checking in to say this is all fine and dandy. bambosh may be somewhat busy but you can always ping me for admin/compat stuff and I'll try to support asap

**Comment by Grafcube** [comment]
I've changed the install path for the icons based on what I saw in other manifest files.

**Comment by Grafcube** [comment]
What is the correct directory to install the icons and desktop file to? I can't seem to find any documentation on this.

**Pull request closed without merging**